opm/parser/eclipse/EclipseState/Aquifer/NumericalAquifer/NumericalAquiferConnection.hpp

```diff
@@ -0,0 +1,73 @@
+/*
+  Copyright (C) 2020 SINTEF Digital
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef OPM_NUMERICALAQUIFERCONNECTION_HPP
+#define OPM_NUMERICALAQUIFERCONNECTION_HPP
+
+#include <opm/parser/eclipse/EclipseState/Grid/FaceDir.hpp>
+#include <map>
+#include <array>
+
+namespace Opm {
+
+    class EclipseGrid;
+    class Deck;
+    class DeckRecord;
+
+    struct NumericalAquiferConnection {
```

Comment:
There is some parts in common with the connection of the two analytical aquifer models (the way to determine whether the face can be connection face), and also some parts are pretty different.  Some refactoring might be done to remove some small duplication code (basically the code of determination of whether a face can be a connection).

Reply:
Preferably, we can do the refactoring in a separate PR. Because that will be involved in refactoring of the existing aquifer models.